vnext/ReactWindowsCore/WebSocketModule.h

```diff
@@ -23,23 +26,46 @@ class WebSocketModule : public facebook::xplat::module::CxxModule
   };
 
   WebSocketModule();
+
+  /// <summary>
+  /// <see cref="facebook::xplat::module::CxxModule::getName" />
+  /// </summary>
   std::string getName();
+
+  /// <summary>
+  /// <see cref="facebook::xplat::module::CxxModule::getConstants" />
+  /// </summary>
   virtual std::map<std::string, folly::dynamic> getConstants();
+
+  /// <summary>
+  /// <see cref="facebook::xplat::module::CxxModule::getMethods" />
+  /// </summary>
+  /// <remarks>See See react-native/Libraries/WebSocket/WebSocket.js</remarks>
   virtual std::vector<Method> getMethods();
 
 private:
+  /// <summary>
+  /// Notifies an event to the current React Instance.
+  /// </summary>
   void SendEvent(std::string&& eventName, folly::dynamic&& parameters);
+
+  /// <summary>
+  /// Creates or retrieves a raw <c>IWebSocket</c> pointer.
+  /// </summary>
   IWebSocket* GetWebSocket(int64_t id, std::string&& url = std::string());
```

Comment:
Is the creation expensive? Should we instead call this method GetOrCreateWebSocket?

Reply:
Sounds reasonable.